tests/HUBViewControllerTests.m

```diff
@@ -91,7 +91,7 @@ @interface HUBViewControllerTests : XCTestCase <HUBViewControllerDelegate>
 @property (nonatomic, assign) BOOL didReceiveViewControllerDidFinishRendering;
 @property (nonatomic, copy) void (^viewControllerDidFinishRenderingBlock)(void);
 @property (nonatomic, copy) BOOL (^viewControllerShouldStartScrollingBlock)(void);
-@property (nonatomic, copy) BOOL (^viewControllerShouldIgnoreHeaderComponentInset)(void);
+@property (nonatomic, copy) BOOL (^viewControllerShouldAutomaticallyManageTopContentInset)(void);
```

Comment:
Just out of curiosity, how come these are blocks rather than just `BOOL` properties?